set.mm

```diff
@@ -409545,17 +409595,18 @@ is a bijection ( 1-1 onto function) in a simple graph.  (Contributed
        Vekens, 12-Oct-2017.)  (Revised by AV, 30-Oct-2020.) $)
     isuvtxa $p |- ( G e. W -> ( UnivVtx ` G ) = { v e. V |
                           A. k e. ( V \ { v } ) E. e e. E { k , v } C_ e } ) $=
-      ( wcel cuvtxa cfv cv cnbgr wral crab cpr wss wrex wa co csn cdif uvtxaval
-      wne wb nbgrel ad2antrr df-3an prcom sseq1i rexbii simpr anim12ci eldifsni
-      w3a eldifi adantl biantrurd syl5rbb syl5bb bitrd ralbidva rabbidva eqtrd
-      jca ) EGJZEKLCMZEAMZNUAJZCFVIUBZUCZOZAFPVHVIQZBMZRZBDSZCVLOZAFPACEFGHUDVG
-      VMVRAFVGVIFJZTZVJVQCVLVTVHVLJZTZVJVHFJZVSTZVHVIUEZVIVHQZVORZBDSZUPZVQVGVJ
-      WIUFVSWABDEVHVIFGHIUGUHWIWDWETZWHTZWBVQWDWEWHUIVQWHWBWKVPWGBDVNWFVOVHVIUJ
-      UKULWBWJWHWBWDWEVTVSWAWCVGVSUMVHFVKUQUNWAWEVTVHFVIUOURVFUSUTVAVBVCVDVE $.
+      ( wcel cuvtxa cfv cv wral crab cpr wss wrex wa syl5bb cnbgr cdif uvtxaval
+      co csn wne w3a nbgrel df-3an prcom sseq1i rexbii eldifi anim12ci eldifsni
+      simpr adantl jca biantrurd syl5rbb ralbidva rabbidva eqtrd ) EGJZEKLCMZEA
+      MZUAUDJZCFVFUEZUBZNZAFOVEVFPZBMZQZBDRZCVINZAFOACEFGHUCVDVJVOAFVDVFFJZSZVG
+      VNCVIVGVEFJZVPSZVEVFUFZVFVEPZVLQZBDRZUGZVQVEVIJZSZVNBDEVEFVFHIUHWDVSVTSZW
+      CSZWFVNVSVTWCUIVNWCWFWHVMWBBDVKWAVLVEVFUJUKULWFWGWCWFVSVTVQVPWEVRVDVPUPVE
+      FVHUMUNWEVTVQVEFVFUOUQURUSUTTTVAVBVC $.
 
     $d E n $.  $d N e k n $.  $d W n $.
-    $( A universal vertex, i.e. an element of the set of all universal
-       vertices.  (Contributed by Alexander van der Vekens, 12-Oct-2017.) $)
+    $( TODO-AV: antecedent ` G e. W ` can be removed!  A universal vertex, i.e.
+       an element of the set of all universal vertices.  (Contributed by
```

Comment:
Missing a verb ?

Reply:
Or is it: "Characterization of a universal vertex" ?

Reply:
Done